playbooks/openshift/README.md

```diff
@@ -76,15 +78,22 @@ Decide a name for your cluster, create a new directory and copy the example conf
     $ cp ~/git/pouta-ansible-cluster/playbooks/openshift/example_cluster_vars.yaml cluster_vars.yaml
 
 Change at least the following config entries:
+
     cluster_name: "YOUR_CLUSTER_NAME" 
     ssh_key: "bastion-key"
-
+    openshift_public_hostname: "your.master.hostname.here"
+    openshift_public_ip: "your.master.ip.here"
```

Comment:
I had a look at a cluster vars file and tried separating variables specific to one OpenShift instance from variables that can apply to multiple instances. Based on that I think the minimum set of variables to configure could also include:
 * project_external_ips
 * bastion_secgroup
 * certificate_crt
 * certificate_key
 * certificate_ca